app/views/bulk_jobs/_bulk_index_table.html.erb

```diff
@@ -14,13 +14,14 @@
         <td><%= job['argo.bulk_metadata.bulk_log_input_file'] %></td>
         <td><%= job['argo.bulk_metadata.bulk_log_note'] %></td>
         <td>
-          <% if(job.has_key?('error')) %>
-            system errors
-          <% elsif(job['argo.bulk_metadata.bulk_log_job_complete']) %>
+          <% if(job['argo.bulk_metadata.bulk_log_job_complete']) %>
             completed
           <% else %>
             in process
           <% end %>
+          <% if(job.has_key?('error')) %>
```

Comment:
Should this block be nested in the block on line 18?  Would anything ever be "In process with system errors"?

Reply:
> Would anything ever be "In process with system errors"?

Yes, apparently that is possible and that is the gist of the ticket that spawned this work (#2068) ... sorry forgot to add to the PR description originally.

I presume what is happening is that some of the rows in the spreadsheet succeed and some do not, those the system errors state while still in progress.

Reply:
Can you write tests for these cases? You may consider doing a ViewComponent, as it's easier to test, but not required.

Reply:
i did a bit of refactoring of the view (but not a component), add a new status, and then added a test to the existing view test to test the status display